lib/player/player_types.rb

```diff
@@ -0,0 +1,9 @@
+module PlayerType
+    def human
+        "H"
```

Comment:
Why use the string labels? Why not just have a `HumanPlayer` class and a `ComputerPlayer` class?

Reply:
this is the valid inputs for when setting a player type before game play. 
I use "H" and "C" a few places in the code , including tests, so I kept them here.  is that ok?